v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/transforms/SourceWriterFn.java

```diff
@@ -90,48 +92,59 @@ public class SourceWriterFn extends DoFn<KV<Long, TrimmedShardedDataChangeRecord
   private static final Distribution UNSUCCESSFUL_WRITE_LATENCY_MS =
       Metrics.distribution(SourceWriterFn.class, "unsuccessful_write_to_source_latency_ms");
 
-  private final ISchemaMapper schemaMapper;
   private final String sourceDbTimezoneOffset;
   private final List<Shard> shards;
   private final SpannerConfig spannerConfig;
   private transient SpannerDao spannerDao;
-  private final Ddl ddl;
-  private final Ddl shadowTableDdl;
   private final SourceSchema sourceSchema;
   private final String shadowTablePrefix;
   private final String skipDirName;
   private final int maxThreadPerDataflowWorker;
   private final String source;
-  private SourceProcessor sourceProcessor;
+  private transient SourceProcessor sourceProcessor;
```

Comment:
In general its dangerous to make these unrelated changes in a PR targeting a specific feature...was there a reason you made this change?